specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json

```diff
@@ -64,7 +64,7 @@
               "format": "file"
             },
             "required": true,
-            "description": "The file contents to include when appending to the file."
+            "description": "The file contents to include when appending to the file.  The maximum content size is 4MB.  For content larger than 4MB you must append the content in 4MB chunks."
```

Comment:
where did we got this from ?

Reply:
I've also asked somebody from the team to review.